README.md

```diff
@@ -1 +1,284 @@
\ No newline at end of file
+# ✨ Macrometa Graph Example with JavaScript SDK ✨
+
+## 👀 Overview
+
+This example demonstrates the ability of graph databases in analyzing and
+offering recommendations for items that are commonly purchased together. By
+establishing connections between customer and item vertices using edge
+collections, the graph is able to seamlessly and accurately deliver tailored
+recommendations to customers who have bought a particular item.
+
+![graph-image](/images/graph-image.png)
+
+In the graph that we created customers Dino and Durga bought the same items. The
+graph is able to recommend items to Dallas based on the items that Dino and
+Durga bought. This is a simple example of how graph databases can be used to
+offer recommendations to customers.
+
+## ❓ How to use this demo
+
+You can use this demo to understand how to use Macrometa SDK to create graph
+database and perform operations on it. In sample data folder you will find
+sample data that you can use to import in Macrometa GDN. We suggest that you
+follow steps explained in [Graph data setup](#graph-data-setup) section to import data and
+create graph. After that you can interact with graph using scripts in `create`,
+`read`, `update` and `delete` folders. (CRUD operations on graph)
+
+## 🚀 Run it Locally
+
+### Prerequisites
+
+1. Create a free [Macrometa account](https://auth-play.macrometa.io/sign-up)
+2. You will have to have **Node.js 14 (or later)** installed in your system. Run
+   the following command in your CLI to check your Node.js version.
+
+```
+node -v
+```
+
+### Initial setup
+
+1. Open a CLI and run the command below to clone this repository. You can do:
+
+```
+git clone https://github.com/Macrometacorp/macrometa-graph-example.git
+```
+
+or
+
+```
+git clone git@github.com:Macrometacorp/macrometa-graph-example.git
+```
+
+depending on your GitHub authentication method.
+
+2. Create a new `.env` file and add your environment variables on your selected
+   path.  
+   Here is how `.env` file should look like:
+
+```
+BASE_URL=https://play.paas.macrometa.io/
+API_KEY=<your-api-key>
+FABRIC=<your-fabric-name> (default: _system)
+```
+
+**Note**: `.env.sample` is only a sample file that you can also use as a guide.
+
+3. Install the project dependencies with:
+
+```
+npm install
+```
+
+### Graph data setup
+
+1. Create two document collections in Macrometa GDN using Macrometa console.
+
+   - `groceryItems`
+   - `groceryCustomers`
+
+2. Create one edge collection in Macrometa GDN using Macrometa console.
+
+   - `groceryOrders`
+
+3. Import test data from `sample-data` folder in Macrometa GDN.
+
+4. Import `sample-data/import-query.json` in Macrometa GDN using Macrometa
+   console.
+
+Example how to import data in Macrometa GDN using Macrometa console:
+![Create Collection](/images/create-collection.png)
+
+![Import Data](/images/import-data.png)
+
+
+
+Example how to import query in Macrometa GDN using Macrometa console:
```

Comment:
That is a good point, but the drawback is that users need to install CLI and do some settings to begin with. It could be more straightforward forward to use console import. For just a test, perhaps it is easier. I just wanted to let you know that it is up for discussion.